packages/Material-Tools.package/MDThemeConfigurationWindow.class/instance/buildColorPickerAndBrightnessButtons.st

```diff
@@ -1,13 +1,22 @@
 initialization
-buildBrightnessButtons
+buildColorPickerAndBrightnessButtons
```

Comment:
This method got pretty long. Maybe we can separate the creation of color & brightness sections?